packages/terra-menu/src/terra-dev-site/test/menu/menu/DynamicMenuChildren.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
```

Comment:
Created this test page to test `this.props.children.length !== prevProps.children.length` logic in `Menu.jsx`